bundles/net.tourbook/src/net/tourbook/database/ITourDataUpdate.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (C) 2022 Frédéric Bard
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package net.tourbook.database;
+
+import net.tourbook.data.TourData;
+
+/**
+ * Interface to update tour values for {@link TourData}
+ */
+public interface ITourDataUpdate {
+
+   public int getDatabaseVersion();
+
+   public String getSplashManagerMessage();
+
+   public void updateTourData(TourData tourData);
```

Comment:
I think that `updateTourData(...)` should return a boolean, when `true` the tour is saved afterwards, when `false` tour is not saved.

I just saw this improvement when I looked at your `TourDataUpdate_047_to_048.updateTourData` method

Reply:
Good idea, it could make the upgrade faster.

Fixed